src/app/BufferedReadCallback.cpp

```diff
@@ -65,9 +65,9 @@ CHIP_ERROR BufferedReadCallback::GenerateListTLV(TLV::ScopedBufferTLVReader & aR
     // To avoid that, a single contiguous buffer is the best likely approach for now.
     //
     uint32_t totalBufSize = 0;
-    for (size_t i = 0; i < mBufferedList.size(); i++)
+    for (auto & i : mBufferedList)
```

Comment:
That's really odd variable naming...  And does this need to be a non-const ref?

Reply:
Automated tool seems limited. Will  manually go through things.